modules/core/assets/items/clips/railClip/railClip.json

```diff
@@ -1,7 +1,7 @@
 {
     "projectile": "rail",
-    "size": 5,
-    "price": 70,
+    "size": 200,
+    "price": 20,
```

Comment:
As I mention before, I see no reason to change these guns in core.

Is there any particular reason for doing this, that I'm missing?